glib/sys/Gir.toml

```diff
@@ -68,49 +68,34 @@ name = "GLib.*"
 status = "generate"
     [[object.function]]
     pattern = "filename_(to|from)_(uri|utf8)"
-    is_windows_utf8 = true
     [[object.function]]
     pattern = "(g|s|uns)etenv"
-    is_windows_utf8 = true
     [[object.function]]
     name = "file_test"
-    is_windows_utf8 = true
     [[object.function]]
     name = "file_get_contents"
-    is_windows_utf8 = true
     [[object.function]]
     name = "file_open_tmp"
-    is_windows_utf8 = true
     [[object.function]]
     name = "mkstemp"
-    is_windows_utf8 = true
     [[object.function]]
```

Comment:
all of those empty configurations should be dropped.

Reply:
Should I return the `is_windows_utf8 = true`  for the `mkstemp` function here?

Reply:
No, you should remove them because they're empty now and as such have no effect.

Except for the ones that need to stay `manual = true` because they need custom handling, or `string_type = "os_string"` because they should be `OsString` instead of `PathBuf`. Those would only have exactly that configuration and nowhere you would keep `is_windows_utf8 = true` as that's meaningless now.